src/helpers/internal_viewport.ts

```diff
@@ -23,8 +13,8 @@ export class InternalViewport {
   offsetScrollbarY: Pixel;
   canScrollVertically: boolean;
   canScrollHorizontally: boolean;
-  width: Pixel;
-  height: Pixel;
+  clientWidth: Pixel;
```

Comment:
When I discussed this with Adrien, it turns out he did not really understand the meaning of that prop. the constructor argument `sizeInGrid` is a bit more explicit but yet fails to really describe what we want to; 
It is the visible size (width/height) of the viewport inside the grid, hence the idea to reuse the term clientWidth,  maybe visibleWidth/visibleHeight would be better ?

Reply:
we are definitely mixing the two concepts. the reason we did this when introducing the frozen pane is that the concept of pane size seemed to only make sense for some viewports and not the others (it's not 100%  but in a real life situation, it is)
And since we only display the content of the viewport, we decided to gowith the vieport naming.

note:: while we named the class viewport, is HAS to be aware of it maximum size because of the scrolling stuff

A more accurate way to describ things would be to stick with the "Pane" as we define them in the core data and have the pane expose a viewport (for instance to use the getVisibleRect getters). but the sutation would be reversed. the Pane would HAVE to be aware of its visible part in the client (DOM) viewport.   Do you think it's more appropriate ? (I kinda do tbh)